src/components/PlatformOnboarding/Steps/LinkAccountsConfirmationStep.tsx

```diff
@@ -0,0 +1,139 @@
+import { useContext } from 'react'
+import { HStack, VStack } from '../../ui/Stack/Stack'
+import { Heading } from '../../ui/Typography/Heading'
+import { P } from '../../ui/Typography/Text'
+import { LinkedAccountsContext } from '../../../contexts/LinkedAccountsContext'
+import { ConditionalList } from '../../utility/ConditionalList'
+import { LinkedAccountToConfirm } from '../../LinkedAccounts/ConfirmationModal/LinkedAccountToConfirm'
+import { useForm } from '@tanstack/react-form'
+import { getAccountsNeedingConfirmation } from '../../../hooks/useLinkedAccounts/useLinkedAccounts'
+import { Button, ButtonVariant } from '../../Button'
+import { useWizard } from '../../Wizard/Wizard'
+import { useConfirmAndExcludeMultiple } from '../../LinkedAccounts/ConfirmationModal/useConfirmAndExcludeMultiple'
+import { LinkAccountsListContainer } from '../Container/LinkAccountsListContainer'
+
+function getSubmitButtonText({
+  totalCount,
+  confirmedCount,
+}: { totalCount: number, confirmedCount: number }) {
+  if (confirmedCount === totalCount) {
+    return totalCount > 1
+      ? 'Confirm All Accounts'
+      : 'Confirm Account'
+  }
+
+  if (confirmedCount === 0) {
+    return totalCount > 1
+      ? 'Exclude All Accounts'
+      : 'Exclude Account'
+  }
+
+  return `Confirm ${confirmedCount} Selected Account${confirmedCount > 1 ? 's' : ''}`
+}
+
+function AccountConfirmationEmptyList() {
+  const CLASS_NAME = 'Layer__AccountConfirmationEmptyList'
+
+  return (
+    <div className={CLASS_NAME}>
+      <VStack slot='center' gap='xs'>
+        <Heading size='sm' align='center'>No Accounts Need Confirmation</Heading>
+        <P variant='subtle' align='center'>You are free to skip this step!</P>
+      </VStack>
+    </div>
+  )
+}
+
+export function LinkAccountsConfirmationStep() {
+  const {
+    data: linkedAccounts,
+    loadingStatus: linkedAccountsLoadingStatus,
+    refetchAccounts,
+  } = useContext(LinkedAccountsContext)
+
+  const effectiveLinkedAccounts = linkedAccounts
+    ? getAccountsNeedingConfirmation(linkedAccounts)
+    : []
+
+  const { trigger } = useConfirmAndExcludeMultiple({ onSuccess: refetchAccounts })
+
+  const { previous, next } = useWizard()
+
+  const {
+    Field,
+    Subscribe,
+    // eslint-disable-next-line @typescript-eslint/unbound-method
+    handleSubmit,
+  } = useForm({
+    defaultValues: {
+      accounts: effectiveLinkedAccounts.map(account => ({ account, isSelected: true })),
+    },
+    onSubmit: async ({ value }) => {
+      const formattedArg = Object.fromEntries(
+        value.accounts.map(({ account, isSelected }) => [account.id, isSelected]),
+      )
+
+      await trigger(formattedArg)
+      await next()
+    },
+  })
+
+  return (
+    <VStack gap='lg'>
+      <VStack gap='2xs'>
+        <Heading level={3} size='sm'>
+          Which accounts do you use for businesses?
+        </Heading>
+        <P variant='subtle'>
+          Please unselect any accounts you don&apos;t use for your business.
```

Comment:
Can we not use quoted strings to avoid having to put in ugly html escape characters? I do not like this and don't like the risk of it leaking as raw text in the front end in the future